_data/simple-icons.json

```diff
@@ -9844,7 +9850,7 @@
         {
             "title": "Vue.js",
             "hex": "4FC08D",
-            "source": "https://github.com/vuejs/art",
+            "source": "https://github.com/vuejs/art/blob/master/logo.svg",
```

Comment:
Should be https://github.com/vuejs/art/blob/a1c78b74569b70a25300925b4eacfefcc143b8f6/logo.svg